src/views/kbv/self-assessment-payment.njk

```diff
@@ -1,9 +1,21 @@
 {% extends "base-form.njk" %}
 {% from "hmpo-html/macro.njk" import hmpoHtml %}
+{% from "components/onPageLoad/macro.njk" import ga4OnPageLoad %}
 
 {% set gtmJourney = "self-assessment-payment - start" %}
 
 {% block mainContent %}
   {{ super() }}
   {{ hmpoHtml(translate("pages.shared.abandon")) }}
+
+  {{ ga4OnPageLoad({
+    nonce: cspNonce,
+    statusCode: '200',
+    englishPageTitle: 'pages.questionEnterRecentSelfAssessmentPayment.title' | translate,
```

Comment:
I think we might be able to get the page key through a template var? rather than hardcoding